src/locales/en-us/common/index.yaml

```diff
@@ -10,6 +10,19 @@ common:
     mode:
       standalone: Standalone
       global: Multi-Zone
+  warnings:
+    CA_EXPIRED: !!text/markdown |
+      The CA for this dataplane has expired
+    STORE_TYPE_MEMORY: !!text/markdown |
+      When using the `memory` store type the state is not persisted. This store should only be used when trying things out. <a target="_blank" href="{KUMA_DOCS_URL}/documentation/configuration/#store">Read more about store types</a>
+    INCOMPATIBLE_UNSUPPORTED_ENVOY: !!text/markdown |
+      Envoy (**{ envoy }**) is unsupported by the current version of Kuma DP (**{ kumaDp }**)
+    INCOMPATIBLE_UNSUPPORTED_KUMA_DP: !!text/markdown |
+      Unsupported version of Kuma DP (**{ kumaDp }**)
+    INCOMPATIBLE_ZONE_CP_AND_KUMA_DP_VERSIONS: !!text/markdown |
+      There is a mismatch between versions of Kuma DP (**{ kumaDp }**) and the Zone Control Plane.
+    INCOMPATIBLE_ZONE_AND_GLOBAL_CPS_VERSIONS: !!text/markdown |
+      There is mismatch between versions of Zone Control Plane (**{ zoneCpVersion }**) and the Global Control Plane (**{ KUMA_VERSION }**)
```

Comment:
The uppercase here is a little awkward, but its just a styling thing, and I've already changed quite a bit. I might alter that at some point in the future.